pkg/api/serialization_proto_test.go

```diff
@@ -38,7 +38,7 @@ import (
 
 func init() {
 	codecsToTest = append(codecsToTest, func(version unversioned.GroupVersion, item runtime.Object) (runtime.Codec, error) {
-		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme))
+		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme), "application/vnd.kubernetes.protobuf")
```

Comment:
Since this is main proto-serialization-related test, can you please use this ContentTypeProtobuf constant instead (I know it's not that important in tests, but it would make me feel more comfortable :))